LFS171x/docs/introduction-to-hyperledger-fabric.md

```diff
@@ -415,7 +415,7 @@ If you need further details on these prerequisites, visit Chapter 4, _Technical
 Next, we will download the latest released Docker images for Hyperledger Fabric, and tag them with the **latest** tag. Execute the command from within the directory into which you will extract the platform-specific binaries:
 
 ```
-$ curl -sSL https://goo.gl/6wtTN5 | bash -s 1.1.0
+$ curl -sSL https://goo.gl/6wtTN5 | bash -s 1.2.0
```

Comment:
Latest command is this
`curl -sSL http://bit.ly/2ysbOFE | bash -s 1.3.0`

Have we verified that the code works with 1.3.0?

Reply:
Just tried that today and it seems to work.  

TO DO after the next comment from @tkuhrt is resolved: 
Also update the link to go find the new URL to https://hyperledger-fabric.readthedocs.io/en/release-1.3/install.html and the screenshot of docker images to 
![image](https://user-images.githubusercontent.com/10833935/47121638-4997a580-d228-11e8-8744-076e45ee1da2.png)

Reply:
@DBAti, you mentioned in today's Training & Education Working Group call that you hit issues with the 1.3 Fabric stuff?  I just went to the end of the 1st example with the 1.3.0 instruction above that Tracy gave.  I didn't have time to go through entire chapter with 1.3.  Please could you let us know the details of what you encountered?

Reply:
@nathalie-ckc @tkuhrt - let event_hub = fabric_client.newEventHub(); this is the function creating the problem with new fabric-client 1.3.0, as I know for now.
It needs to be changed to let event_hub = channel.newChannelEventHub('localhost:7051');